ui/page/root/overview_page.go

```diff
@@ -386,7 +386,7 @@ func (pg *OverviewPage) assetBalanceItemLayout(item assetBalanceSliderItem) layo
 		return pg.sliderRedirectBtn.Layout(gtx, func(gtx C) D {
 			return layout.Stack{}.Layout(gtx,
 				layout.Stacked(func(gtx C) D {
-					return item.backgroundImage.LayoutSize2(gtx, unit.Dp(gtx.Constraints.Max.X), values.MarginPadding221)
+					return item.backgroundImage.LayoutSizeWithRadius(gtx, unit.Dp(gtx.Constraints.Max.X), values.MarginPadding221, 8)
```

Comment:
Please change `item assetBalanceSliderItem` to a pointer (`item *assetBalanceSliderItem`). `pg.assetBalanceSliderLayout()` should also use pointers. Unless there's a good reason to prefer values over pointers.